new_solution.py

```diff
@@ -70,25 +164,801 @@ def _convert_step1(self):
+                        )
+                    extract[record_counter]['objectType'] = \
+                            vehicle_desc_dict[vehicle_id] + ' (стара форма)'
+                    extract[record_counter]['rights'][owner_id] = \
+                            self._convert_using_rules(
+                                [
+                                 (None, 'citizen', ""),
+                                 (None, 'ua_city', ""),
+                                 (None, 'ua_street', ""),
+                                 (None, 'ua_lastname', ""),
+                                 (None, 'ua_postCode', ""),
+                                 (None, 'eng_lastname', ""),
+                                 (None, 'eng_postCode', ""),
+                                 (None, 'rightBelongs', owner_id),
+                                 (None, 'ua_firstname', ""),
+                                 (None, 'ukr_lastname', ""),
+                                 (None, 'eng_firstname', ""),
+                                 (None, 'ownershipType', ""),
+                                 (None, 'ua_middlename', ""),
+                                 (None, 'ua_streetType', ""),
+                                 (None, 'ukr_firstname', ""),
+                                 (None, 'eng_middlename', ""),
+                                 (None, 'otherOwnership', ""),
+                                 (None, 'ukr_middlename', ""),
+                                 (None, 'rights_cityPath', ""),
+                                 (None, 'ua_company_name', ""),
+                                 (None, 'eng_company_name', ""),
+                                 (None, 'ukr_company_name', ""),
+                                 (None, 'percent-ownership', '100'),
+                                 (None, 'ua_street_extendedstatus', ""),
+                                 (None, 'ua_houseNum_extendedstatus', ""),
+                                 (None, 'ua_postCode_extendedstatus', ""),
+                                 (None, 'eng_postCode_extendedstatus', ""),
+                                 (None, 'ua_middlename_extendedstatus', ""),
+                                 (None, 'eng_middlename_extendedstatus', ""),
+                                 (None, 'ukr_middlename_extendedstatus', ""),
+                                 (None, 'ua_housePartNum_extendedstatus', ""),
+                                 (None, 'ua_apartmentsNum_extendedstatus', "")
+                                ],
+                                vehicle
+                    )
+                    record_counter += 1
+        
+        return {
+            "step_6": extract
+        }
+
+    def _convert_step7(self):
+        extract = {}
+        record_counter = 1
+        papers_desc = 'Номінальна вартість цінних паперів  (стара форма)'
+        papers_info = {}
+        temp_dict = self._jsrch('banks')
+        # self._jsrch('banks.47') - does not work - jmespath.exceptions.ParseError
```

Comment:
>>> import jmespath
>>> a = {'banks': {47: 'bla'}}
>>> jmespath.search('banks."47"', a)
>>> jmespath.search('banks.47', a)
jmespath.exceptions.ParseError: Expecting: ['quoted_identifier', 'unquoted_identifier', 'lbracket', 'lbrace'], got: number: Parse error at column 6, token "47" (NUMBER), for expression:
"banks.47"